Patch to install ISC python library files to vendor-packages instead
of site-packages directory.

--- old/./configure.in	2018-09-04 11:18:50.810670608 -0700
+++ new/./configure.in	2018-09-04 11:18:50.606344674 -0700
@@ -310,6 +310,16 @@
 			case "$use_python" in
 			/*)
 				PYTHON="$use_python"
+				case "$use_python_install_dir" in
+				unspec)
+					PYTHON_INSTALL_DIR=""
+					PYTHON_INSTALL_LIB=""
+					;;
+				*)
+					PYTHON_INSTALL_DIR="$use_python_install_dir"
+					PYTHON_INSTALL_LIB="--install-lib=$use_python_install_dir"
+					;;
+				esac
 				;;
 			*)
 				AC_PATH_PROGS(PYTHON, $use_python)
